calm_adapter/calm_adapter/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+    <logger name="org.apache.http" level="WARN"/>
+    <logger name="org.eclipse.jetty" level="WARN"/>
+  <root level="INFO">
+    <appender-ref ref="STDOUT" />
+  </root>
+</configuration>
```

Comment:
Any reason we don't use the logstash stuff here?
https://github.com/wellcometrust/catalogue/blob/master/api/api/src/main/resources/logback.xml#L19-L22

Reply:
This is for the tests

Reply:
logstash setup already comimitted, does this look sufficient?
https://github.com/wellcometrust/catalogue/blob/master/calm_adapter/calm_adapter/src/main/resources/logback.xml#L8-L17